app/views/case_information/_form.html.erb

```diff
@@ -21,7 +21,7 @@
 
         <div class="govuk-radios">
           <div class="govuk-radios__item">
-            <%= radio_button_tag("case_information[welsh_offender]", 'Yes', @case_info.welsh_offender == 'Yes', class: "govuk-radios__input") %>
+            <%= radio_button_tag("case_information[welsh_offender]", 'Yes', @case_info.welsh_offender == 'Yes', class: "govuk-radios__input", id: 'welsh_offender') %>
```

Comment:
This is okay, but I'd be tempted to put the id="welsh-offender" on the previous line (in the div). And same for the following ones, but it's up to you.